cmd/server/pactasrv/initiative.go

```diff
@@ -0,0 +1,164 @@
+package pactasrv
+
+import (
+	"context"
+
+	"github.com/RMI/pacta/db"
+	api "github.com/RMI/pacta/openapi/pacta"
+	"github.com/RMI/pacta/pacta"
+)
+
+// Creates a initiative
+// (POST /initiatives)
+func (s *Server) CreateInitiative(ctx context.Context, request api.CreateInitiativeRequestObject) (api.CreateInitiativeResponseObject, error) {
+	err := s.createInitiative(ctx, request)
+	if err != nil {
+		e := errToAPIError(err)
+		return api.CreateInitiativedefaultJSONResponse{
+			Body:       e,
+			StatusCode: int(e.Code),
+		}, nil
+	}
+	return api.CreateInitiative200JSONResponse(emptySuccess), nil
```

Comment:
I don't think you should need any of this `emptysucess` stuff, an empty struct should be sufficient, e.g.

```suggestion
	return api.CreateInitiative200JSONResponse{}, nil
```

Reply:
GC done.